cmd/virt-handler/virt-handler.go

```diff
@@ -286,6 +290,11 @@ func (app *virtHandlerApp) Run() {
 		0,
 		cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
 
+	lwPVC := cache.NewListWatchFromClient(app.virtCli.RestClient(), "persistentvolumeclaims", k8sv1.NamespaceAll, fields.Everything())
+	pvcInformer := cache.NewSharedIndexInformer(lwPVC, &k8sv1.PersistentVolumeClaim{}, app.PVCWatchTimeoutDuration, cache.Indexers{cache.NamespaceIndex: cache.MetaNamespaceIndexFunc})
```

Comment:
why not `factory.PersistentVolumeClaim()`?

Reply:
I wanted to reduce the resync period

Reply:
I'm not sure it is necessary to raise it but 24 hours seemed high (I thought the test flakiness is due to not syncing, but it was something else).

Reply:
increasing resync interval mostly serves to mask controller bugs so not the best way to cure flakiness.  24hrs is quite common resync period